pandas/io/json/json.py

```diff
@@ -11,8 +11,13 @@
 
 from pandas.core.dtypes.common import is_period_dtype
 
-from pandas import DataFrame, MultiIndex, Series, compat, isna, to_datetime
+from pandas import compat
+from pandas.core.frame import DataFrame
+from pandas.core.index import MultiIndex
+from pandas.core.missing import isna
 from pandas.core.reshape.concat import concat
```

Comment:
@WillAyd I think it would still be useful to have this PR as there are a few places which would still not be importing from top level if everything was reverted back .....like here